builds/mariner/SPECS/aziot-edge/aziot-edge.spec

```diff
@@ -0,0 +1,189 @@
+%define iotedge_socketdir %{_localstatedir}/lib/iotedge
+%define aziot_confdir %{_sysconfdir}/aziot
+%define iotedge_confdir %{aziot_confdir}/edged
+
+Name:           aziot-edge
+Version:        @@VERSION@@
+Release:        3%{?dist}
+
+License:        MIT
+Summary:        Azure IoT Edge Module Runtime
+URL:            https://github.com/azure/iotedge
+
+%{?systemd_requires}
+BuildRequires:  systemd
+BuildRequires:  rust >= 1.47.0
+Requires(pre):  shadow-utils
+Requires:       openssl
+Requires:       moby-engine
+Requires:       moby-cli
+
+#Source0:       https://github.com/Azure/iotedge/archive/%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+
+%description
+Azure IoT Edge Module Runtime
+Azure IoT Edge is a fully managed service that delivers cloud intelligence
+locally by deploying and running artificial intelligence (AI), Azure services,
+and custom logic directly on cross-platform IoT devices. Run your IoT solution
+securely and at scale—whether in the cloud or offline.
+
+This package contains the IoT Edge daemon and CLI tool.
+
+%prep
+%setup -q
+
+%build
+IOTEDGE_HOST=unix:///var/lib/iotedge/mgmt.sock
+export IOTEDGE_HOST
+
+cd edgelet
+make \
+    CONNECT_MANAGEMENT_URI=unix://%{iotedge_socketdir}/mgmt.sock \
+    CONNECT_WORKLOAD_URI=unix://%{iotedge_socketdir}/workload.sock \
+    LISTEN_MANAGEMENT_URI=unix://%{iotedge_socketdir}/mgmt.sock \
+    LISTEN_WORKLOAD_URI=unix://%{iotedge_socketdir}/workload.sock \
+    release
+
+%install
+IOTEDGE_HOST=unix:///var/lib/iotedge/mgmt.sock
+export IOTEDGE_HOST
+
+rm -rf $RPM_BUILD_ROOT
+cd edgelet
+make \
+    CONNECT_MANAGEMENT_URI=unix://%{iotedge_socketdir}/mgmt.sock \
+    CONNECT_WORKLOAD_URI=unix://%{iotedge_socketdir}/workload.sock \
+    LISTEN_MANAGEMENT_URI=unix://%{iotedge_socketdir}/mgmt.sock \
+    LISTEN_WORKLOAD_URI=unix://%{iotedge_socketdir}/workload.sock \
+    DESTDIR=$RPM_BUILD_ROOT \
+    unitdir=%{_unitdir} \
+    docdir=%{_docdir}/%{name} \
+    install
+
+%clean
+rm -rf $RPM_BUILD_ROOT
+
+%pre
+# Check for container runtime
+if ! /usr/bin/getent group docker >/dev/null; then
+    echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+    echo ""
+    echo " ERROR: No container runtime detected."
+    echo ""
+    echo " Please install a container runtime and run this install again."
+    echo ""
+    echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+
+    exit 1
+fi
+
+# Create iotedge group
+if ! /usr/bin/getent group iotedge >/dev/null; then
+    %{_sbindir}/groupadd -r %{iotedge_group}
+fi
+
+# Create iotedge user
+if ! /usr/bin/getent passwd iotedge >/dev/null; then
+    %{_sbindir}/useradd -r -g %{iotedge_group} -c "iotedge user" -s /bin/nologin -d %{iotedge_home} %{iotedge_user}
+fi
+
+# Add iotedge user to moby-engine group
+if /usr/bin/getent group docker >/dev/null; then
+    %{_sbindir}/usermod -a -G docker %{iotedge_user}
+fi
+
```

Comment:
Please add 
```
# Add iotedge user to systemd-journal group so it can get system logs
if /usr/bin/getent group systemd-journal >/dev/null; then
    %{_sbindir}/usermod -aG systemd-journal %{iotedge_user}
fi
```